packages/api/src/platforms/vtex/clients/commerce/index.ts

```diff
@@ -364,10 +364,12 @@ export const VtexCommerce = (
         'content-type': 'application/json',
       })
 
+      const sessionCookie = parse(ctx?.headers?.cookie ?? '').vtex_session
```

Comment:
se tiver algum cookie mas que não seja o `vtex_session` essa linha não quebra?

Reply:
O `sessionCookie` nesse caso se torna `undefined` e o método `sessionCookie ? 'PATCH' : 'POST'` vai para POST

Reply:
Pelo que testei não deu problema, na primeira chamada não tem vtex_session, que é quando será enviado o POST e ele será criado. Porém, para garantirmos posso adicionar um optional chaining, o que achas?
```suggestion
      const sessionCookie = parse(ctx?.headers?.cookie ?? '')?.vtex_session
```